.github/workflows/upgrade-ci.yml

```
@@ -0,0 +1,158 @@
+  # pull_request:
+  #   branches:
+  #     - main 
+  push:          # Runs on any direct push to any branch 
+
+
+jobs:
+  k8s:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout tackle
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)